### DISCUSSION TITLE
Regarding Zdb

### DISCUSSION THREAD

**Discussion by roopa01** [openzfs/zfs#11168] [Q&A] [answered]
HI,

  I want to use zdb module only to decompress zfs data blocks which is compressed with lz4 compression.
So how can i use this code to build on ubuntu.

**Comment by behlendorf** [comment, marked as answer]
The `zdb` utility should already be installed on Ubuntu.  However, if you need to build a newer version you can build the latest source in the openzfs repository using [this documentation](https://openzfs.github.io/openzfs-docs/Developer%20Resources/Building%20ZFS.html) and run it from the source tree.

```
zdb -R poolname vdev:offset:[<lsize>/]<psize>[:flags]
```
